contracts/ResilientOracle.sol

```diff
@@ -120,7 +135,7 @@ contract ResilientOracle is OwnableUpgradeable, PausableUpgradeable, ResilientOr
     }
 
     /**
-     * @notice Set single token configs, vToken MUST HAVE NOT be added before, and main oracle MUST NOT be zero address
+     * @notice Set/reset single token configs and main oracle MUST NOT be zero address
```

Comment:
```suggestion
     * @notice Set/reset single token configs and main oracle **must not** be zero address
```
What about using bold markdown here so it gets formatted with markdown and we avoid all caps?

Reply:
could still do all caps if we want but bold would be a nice addition